File: fizzbuzz/go/millerbrayden/Fizzbuzz.go
package main

import (
	"fmt"
	"os"
	"strconv"
)

func printWords(n int) {
	for i := 1; i <= n; i++ {
		switch {
		case i%15 == 0:
			fmt.Println("BobCat")
		case i%3 == 0:
			fmt.Println("Bob")
		case i%5 == 0:
			fmt.Println("Cat")
		default:
			fmt.Println(i)
		}
	}
}

func main() {
	if len(os.Args) != 2 {
		fmt.Println("Usage: ./fizzbuzz <number>")
		return
	}
	n, err := strconv.Atoi(os.Args[1])
	if err != nil {
		fmt.Println(err)
		return
	}
	printWords(n)
}

File: fizzbuzz/java/jayce_cripe/FizzBuzz.java
public class FizzBuzz {
    public static void main(String[] args) {
        if (args.length == 0) {
            System.out.println("Please provide a number as an argument.");
            return;
        }

        int n = Integer.parseInt(args[0]);
        System.out.println("Input: n = " + n);

        for (int i = 1; i <= n; i++) {
            if (i % 3 == 0 && i % 5 == 0) {
                System.out.println("BobCat");
            } else if (i % 3 == 0) {
                System.out.println("Bob");
            } else if (i % 5 == 0) {
                System.out.println("Cat");
            } else {
                System.out.println(i);
            }
        }
    }
}